CMakeLists.txt

```diff
@@ -401,7 +401,7 @@ if(NOT HAVE_POSITION_INDEPENDENT_CODE AND NOT WIN32)
 endif()
 
 # Useful additional variables that extensions can use.
-if(UNIX AND NOT APPLE)
+if(CMAKE_SYSTEM MATCHES Linux)
```

Comment:
Nice.

Should this other use of `UNIX AND NOT APPLE` be tweaked too ?

See https://github.com/python-cmake-buildsystem/python-cmake-buildsystem/blob/7812f43ab5cbe2840809997d5d8bbc8b5abd9989/cmake/libpython/CMakeLists.txt#L535